staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -274,7 +274,7 @@ func recordEvent(sink EventSink, event *v1.Event, patch []byte, updateExistingEv
 		klog.Errorf("Unable to construct event '%#v': '%v' (will not retry!)", event, err)
 		return true
 	case *errors.StatusError:
-		if errors.IsAlreadyExists(err) {
+		if errors.IsAlreadyExists(err) || errors.HasStatusCause(err, v1.NamespaceTerminatingCause) {
```

Comment:
Looks like we use this pattern elsewhere:
https://cs.k8s.io/?q=HasStatusCause.*NamespaceTerminatingCause&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

the logic is just making it `Infof` instead of `Errorf`, So i am happy to land this

/approve